docs/release-notes/searcher-context-removal.rst

```diff
@@ -0,0 +1,67 @@
+:orphan:
+
+**Breaking Changes**
+
+-  ASHA: All experiments using ASHA hyperparameter search must now configure ``max_time`` and
+   ``time_metric`` in the experiment config, instead of ``max_length``. Additionally, training code
+   must report the configured ``time_metric`` in validation metrics. As a convenience, Determined
+   training loops now automatically report ``batches`` and ``epochs`` with metrics, which you can
+   use as your ``time_metric``. ASHA experiments without this modification will no longer run.
+
+-  Custom Searchers: all custom searchers including DeepSpeed Autotune were deprecated in ``0.36.0``
+   and are now being removed. Users are encouraged to use a preset searcher, which can be easily
+   :ref:`configured <experiment-configuration_searcher>` for any experiment.
+
+**New Features**
+
+-  API: introduce ``keras.DeterminedCallback``, a new high-level training API for TF Keras that
+   integrates Keras training code with Determined through a single :ref:`Keras Callback
+   <api-keras-ug>`.
+
+-  API: introduce ``deepspeed.Trainer``, a new high-level training API for DeepSpeedTrial that
+   allows for Python-side training loop configurations and includes support for local training.
+
+**Deprecations**
+
+-  Experiment Config: the ``max_length`` field of the searcher configuration section has been
+   deprecated for all experiments and searchers. Users are expected to configure the desired
+   training length directly in training code.
+
+-  Experiment Config: the ``optimizations`` config has been deprecated. Please see :ref:`Training
```

Comment:
Are all these deprecations mirrored in the rest of the docs?  I didn't see optimizations touched in the experiment config reference, for instance.